### PR TITLE
Filter

### PR DESCRIPTION
**Pull request by vermasavita**
What do you need to check?

- Filter notes by date 
- Filter notes by priority

### CODE REVIEW

**Review by Mandhularajitha — APPROVED** [review]
Clean code
Good PR
easy to understand code

File: src/components/filter/filter.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
+.filter-container {
+    box-shadow: rgba(0, 0, 0, 0.16) 0px 1px 4px;
+  position: fixed;
+  left: 60%;
+  bottom: 60%;
+  display: none;
+  flex-direction: column;
```
Comment (RakshaPawar108) [inline src/components/filter/filter.css:7]: I guess you do not need `flex-direction` property if `display `is set to `none`

**Review by sanjitsarkar — APPROVED** [review]
Looked through the code found no issue.
Code is well-written and readable.

**Review by RakshaPawar108 — APPROVED** [review]
Great work Savita! 
Functionality is working as expected
Love the UI

File: src/components/filter/filter.css (hunk 1 above)
Comment (vermasavita) [reply src/components/filter/filter.css:7]: Yes, but it would be needed when user will click on filter button. Coz in that case display will be flex.

**Pull request merged**